CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.1.1
```

Comment:
I think we should reserve patch releases for quick little fixes on major/minor versions.

This can just go into 4.2.0 unless there is a specific reason you need the change urgently

Reply:
Yea I can put it for 4.2.0